sdk/clientcore/optional-dependency-tests/src/test/java/io/clientcore/core/http/pipeline/HttpInstrumentationPolicyTests.java

```diff
@@ -314,27 +311,6 @@ public void tracingIsDisabledOnInstance() throws IOException {
         assertEquals(0, exporter.getFinishedSpanItems().size());
     }
 
-    @Test
-    public void tracingIsDisabledOnRequest() throws IOException {
```

Comment:
removed disabling individual requests - we didn't advertise this feature in azure-core and I don't recall anyone asking for it. 
We also don't support it in other langauges.